### PR TITLE
docs(releasing): minor updates

### PR DESCRIPTION
**Pull request by paulirish**
_update_: see https://github.com/GoogleChrome/lighthouse/pull/5345#issuecomment-424825309

### CODE REVIEW

**Comment by brendankenny** [comment]
wait, it's still not ignored? https://github.com/GoogleChrome/lighthouse/blob/520f9ac06420634ccc03d0326e8e42fe83610a42/.npmignore#L23

**Comment by paulirish** [comment]
@brendankenny blast from the past. ;)

##### changes

* lighthouse-core/scripts wasn't ignored correctly. their syntax is odd if you want most files in a dir ignored but one allowed. this works though.
* keep all of lighthouse-cli/tests. most notably all the fixtures to support smokehouse
* ignore latest-run
* remove mentions of `yarn smokehouse` since [that died here](https://github.com/GoogleChrome/lighthouse/commit/419b023680a51080d078b0c782ebae23b34fefdc).
* remove the ["drive-by-tightening"](https://github.com/GoogleChrome/lighthouse/pull/5025/files#r183588493) that doesnt seem to help and actually prevents the `npm explore lighthouse -- npm run smoke` from working.

File: .npmignore [hunk 1]
```diff
@@ -18,23 +18,17 @@ lighthouse-logger/
...
-!lighthouse-cli/test/smokehouse
-!lighthouse-cli/test/fixtures/
-
-!lighthouse-cli/index.js
-lighthouse-cli/*.map
-lighthouse-cli/commands/*.map
-lighthouse-cli/types/*.js
-lighthouse-cli/types/*.map
+# keep smokehouse tests, etc
+!lighthouse-cli/test
```
Comment (brendankenny) [inline .npmignore]: how much bigger does our package end up?

Comment (paulirish) [reply .npmignore]: oh this was not fun. turns out `yarn pack` builds things VERY differently than `npm pack`. 
and `npm pack` gives basically the same package before and after this PR. wonderful.

Comment (paulirish) [reply .npmignore]: turns out it's smaller now. because the npmignore in master includes all of lighthouse-cli already.

so aint that somethin

**Pull request merged**